code/Caravela.Documentation.SampleCode.AspectFramework/Normalize.Aspect.cs

```diff
@@ -14,7 +14,7 @@ public void BuildAspect( IAspectBuilder<IFieldOrProperty> builder )
         [Template]
         string OverrideProperty
         {
-            set => meta.Target.FieldOrProperty.Value = value?.Trim().ToLowerInvariant();
+            set => meta.Target.FieldOrProperty.Value = value?.Trim()?.ToLowerInvariant();
         }
```

Comment:
I think the additional ? is redundant because it is inferred from the one one value.